app/models/validations/setup_validations.rb

```diff
@@ -42,6 +42,14 @@ def validate_organisation(record)
     end
   end
 
+  def validate_managing_organisation_data_sharing_agremeent_signed(record)
+    return unless FeatureToggle.new_data_protection_confirmation?
+
+    if record.managing_organisation_id_changed? && record.managing_organisation.present? && !record.managing_organisation.data_protection_confirmed?
```

Comment:
should we check if the org is present before we check if the id is changed to avoid a NoMethodError? Or is _changed? happy with that